src/libsync/account.cpp

```diff
@@ -50,6 +58,10 @@ Account::Account(const QUuid &uuid, QObject *parent)
     , _credentialManager(new CredentialManager(this))
 {
     qRegisterMetaType<AccountPtr>("AccountPtr");
+
+    const QString cacheDirectory = QStringLiteral("%1/resources/").arg(determineAccountCacheLocation(this));
+    QDir().mkpath(cacheDirectory);
```

Comment:
Hm should we move that to determineAccountCacheLocation ?

Reply:
Can you also handle the unit test scenario we discussed or open an issue for them

Reply:
> Move the creation of the dir to determineAccountCacheLocation, give it an argument etc.

I guess it will be easier to have a member variable on the account itself that contains the parent directory and initialize it from the constructor.

> Unit tests will use random uuids on every run so we will create hundreds of cache locations on each run.

I'd rather have the cache be enabled all the time. How about a parameter in `Account`'s constructor (and `::create`) that is empty by default but allows for passing a directory explicitly?

Reply:
A static var that caches QStandardPaths::writableLocation(QStandardPaths::CacheLocation) and allows to override it?